dashboard/app/views/pd/application/teacher_application/submitted.html.haml

```diff
@@ -3,19 +3,23 @@
 
 #thanks
   %h1 Thank you for submitting your application!
+  %h2 You will receive confirmation that your application has been submitted at the email used to complete this application.
```

Comment:
@dmcavoy With recent changes that connect the application's alternate email to a user's enrollments, do we want to eventually have the application emails send to their alternate email if one is provided?

Reply:
I think we do use the alternate email but we should double check. I think the ideal if that we send one email with both emails recipients

Reply:
Do we want a change in the wording here or is this good to ship (at least for now)?

Reply:
https://codedotorg.atlassian.net/browse/ACQ-1292